### ISSUE TITLE
cleanupSpans function currupting the pasted html..

### ISSUE THREAD

**Issue by gs-kgopi** [yabwe/medium-editor#1266]
### Description

**cleanupSpans** function removing the styles of copied content.

### Steps to reproduce

1. Try to copy the content which is having a span tag with inline styles (Copy underline words from [here](http://grammar.yourdictionary.com/punctuation/rules-for-underlining.html))
2. Paste the copied content in editor.
3. We can observe the loss of styles (underline in this case)

**Expected behavior:** Ideally the source formatting should not be lost

**Actual behavior:** Styles are getting missed (**cleanupSpans** function is moving the span element content to the it's parent.. while doing so the styles are getting lost)

**Link to an example:** [It's reproducible on the demo page](https://yabwe.github.io/medium-editor/demo.html), 

**Note**: I wan't know, why span element content is getting moved to it's parent element (and removing the span element) ?

### Versions

- medium-editor: Any version
- browser: Not browser related
- OS: Not OS related

**Comment by Olgagr** [comment]
@gs-kgopi This is expected behaviour. Look at [Paste Options](https://github.com/yabwe/medium-editor#paste-options) in the docs. By default, any HTML is pasted as plain text.

**Comment by Olgagr** [comment]
@nmielnik @j0k3r  The same "issue" is reported regularly by different people (see #1226). Maybe we should expose it more in the docs ?

**Comment by gs-kgopi** [comment]
@Olgagr, Thanks for the reply..

I do know, by default plain text will be copied. I don't want that so, made forcePlainText as false.

paste: {
        forcePlainText: false,
        cleanPastedHTML: true
    }

Here the issue is different, cleanupSpans (will be invoked during paste handler) function unwrapping the span elements.. Its causing the issue mentioned.. Please go the ticket description once again..

And let me know.. if you need more information.

**Comment by Olgagr** [comment]
@gs-kgopi Ok, you didn't mention that to reproduce the issue the default settings should be changed. The demo page, you indicated to reproduce the problem, uses the default settings :)

**Comment by nmielnik** [comment]
Ok, so just confirm, the issue here is that the classes on the spans being copied are lost, even when using `forcePlaceText: false` and `cleanPastedHTML: true` in the paste options correct?

**Comment by gs-kgopi** [comment]
@nmielnik Yes, Span elements are getting unwrapped, it leads to loss of element attributes (class, styles and etc..), has impact on copied content styles.
